ztp/gitops-subscriptions/Makefile

```diff
@@ -6,9 +6,14 @@ ACM_POLICYGEN_EX_DIR=./argocd/example/acmpolicygenerator
 ACM_POLICYGEN_KUSTOMIZE_DIR=$(XDG_CONFIG_HOME)/policy.open-cluster-management.io/v1/policygenerator
 POLICYGEN_EX_DIR=./argocd/example/policygentemplates
 POLICYGEN_KUSTOMIZE_DIR=$(XDG_CONFIG_HOME)/ran.openshift.io/v1/policygentemplate
+GITOPS_OPERATOR_SUB_FILE=./argocd/deployment/openshift-gitops-operator.yaml
 KUSTOMIZE_DIR=/tmp
+KUSTOMIZE_VERSION_DIR=kustomize_version_dir
+KUSTOMIZE_VERSION_FILE=./$(KUSTOMIZE_VERSION_DIR)/kustomize_version
 KUSTOMIZE_BIN=$(KUSTOMIZE_DIR)/kustomize
-KUSTOMIZE_VERSION=4.4.0
+KUSTOMIZE_VERSION_DEFAULT=5.2.1
+KUSTOMIZE_GET_VERSION_SCRIPT=../tools/get-kustomize-version/get-gitops-operator-kustomize-version.sh
+KUSTOMIZE_VERSION=$(shell ./$(KUSTOMIZE_GET_VERSION_SCRIPT) v4.16 $(GITOPS_OPERATOR_SUB_FILE) $(KUSTOMIZE_VERSION_DEFAULT) $(KUSTOMIZE_VERSION_DIR) && cat $(KUSTOMIZE_VERSION_FILE))
```

Comment:
I want to see if we can obtain `v4.16` from someplace instead of hardcoding it.

Reply:
May be at least worth putting it in its own variable at the top of the makefile with a note that it needs to be kept up to date?

Reply:
Idea: Check the current git branch; If it's "release-.*" assume `"v${branchname#release-}"` is the OCP version.  Otherwise fall back to some hard-coded default?